test/forms/AppointAgentRequestSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import models.propertyrepresentation.{AppointAgentRequest, AppointmentScope}
+import org.scalatest.{FlatSpec, MustMatchers}
+import utils.FormBindingVerification._
+import views.helpers.Errors
+
+class AppointAgentRequestSpec extends FlatSpec with MustMatchers {
```

Comment:
FlatSpec is not our standard.

Reply:
I am now extending VoaPropertyLinkingSpec. It still used FlatSpec 🤷‍♂